.github/workflows/ci_e2e.yaml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Tests
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: '53 0 * * SAT' # Every Saturday at 00:53 UTC
```

Comment:
**non-blocking/question:** Just for my learning, why 00:53AM? To avoid low resource availability running at usual midnight?

Reply:
yes that was the idea